<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechOutputSetupUI</class>
 <widget class="QDialog" name="QtArteryTechOutputSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechOutputSetupUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="RefMonitorPointLabel">
         <property name="text">
          <string>Ref Monitor Point:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RefMonitorPointLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="MonitorPoint1Label">
         <property name="text">
          <string>Monitor Point 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MonitorPoint1LineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="MonitorPoint2Label">
         <property name="text">
          <string>Monitor Point 2: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MonitorPoint2LineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="MonitorPoint3Label">
         <property name="text">
          <string>Monitor Point 3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MonitorPoint3LineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="FFRMonitorFileLabel">
         <property name="text">
          <string>FFR Monitor File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FFRMonitorFileLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="FlowRateMonitorFileLabel">
         <property name="text">
          <string>Flow Rate Monitor File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FlowRateMonitorFileLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="SolutionOutputFileLabel">
         <property name="text">
          <string>Solution Output File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SolutionOutputFileLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="ShowSubdomainMeshDataLabel">
         <property name="text">
          <string>Show Subdomain Mesh Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ShowSubdomainMeshDataLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="ShowMeshDataLabel">
         <property name="text">
          <string>Show Mesh Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ShowMeshDataLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="ShowSolutionLabel">
         <property name="text">
          <string>Show Solution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ShowSolutionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="MonitorResidualLabel">
         <property name="text">
          <string>Monitor Residual:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MonitorResidualLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="NumberOfStepsForOutputLabel">
         <property name="text">
          <string>Number of Steps for Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NumberOfStepsForOutputLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="TimeIntervalForOutputLabel">
         <property name="text">
          <string>Time Interval for Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TimeIntervalForOutputLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="OkPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
